### PR TITLE
[patch] fix db2_backup role (var name mismatch)

### PR DESCRIPTION
**Pull request by cuddlyporcupine**
- Bug fix: mismatch variable name for db2 pod
- Bug fix: backup was written to ephemeral storage inside db2 pod, resulting in pod restart for large database backups.  backup is now written to the backup pvc of the db2 pod

### CODE REVIEW

File: ibm/mas_devops/roles/db2_backup/defaults/main.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # Db2 backup default vars
 db2_backup_dir: "{{ lookup('env', 'DB2_BACKUP_DIR') }}"
 db2_backup_instance_name: "{{ lookup('env', 'DB2_BACKUP_INSTANCE_NAME') }}"
+db2_namespace: "{{ lookup('env', 'DB2_NAMESPACE') }}"
```
Comment (alequint) [inline ibm/mas_devops/roles/db2_backup/defaults/main.yml:5]: @cuddlyporcupine  To be consistent with default values used in db2 role, I would suggest we default namespace in backup/restore roles with same value as [here](https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/db2/defaults/main.yml#L4)

File: ibm/mas_devops/roles/db2_backup/README.md [hunk 2]
```diff
@@ -33,6 +39,8 @@ Example Playbook
 - hosts: localhost
   any_errors_fatal: true
   vars:
+    db2_namespace: "db2u"
```
Comment (alequint) [inline ibm/mas_devops/roles/db2_backup/README.md:42]: @cuddlyporcupine  To be consistent with default values used in db2 role, I would suggest we default namespace in backup/restore roles with same value as [here](https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/db2/defaults/main.yml#L4)

**Review by alequint — CHANGES_REQUESTED** [review]

File: ibm/mas_devops/roles/db2_backup/defaults/main.yml (hunk 1 above)
Comment (cuddlyporcupine) [reply ibm/mas_devops/roles/db2_backup/defaults/main.yml]: @alequint requested change for default is in

**Review by alequint — APPROVED** [review]

**Comment by durera** [comment]
Also, can you write up the summary description for this PR please .. doesn't need an essay, just a paragraph (or even a sentence) sumarising what is being changed and why.

**Review by alequint — DISMISSED** [review]

**Comment by alequint** [comment]
@cuddlyporcupine dicker-build action is failing, but not sure why... happening only in this PR/branch. Can you check?

**Pull request closed without merging**